mlir/include/mlir/Dialect/MIOpen/utility/common.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef COMMON_H
+#define COMMON_H
+struct ArgumentFields {
```

Comment:
please move the comment of `ArgumentFields` here.

Reply:
done